include/frg/logging.hpp

```diff
@@ -135,6 +135,42 @@ constexpr auto output_to(Container &cont) {
 	return container_logger{cont};
 }
 
+#ifdef FRIGG_HAS_IOSTREAM
+	template<typename Out>
```

Comment:
If this is generic, to() should maybe be also?  with a concept to check for `op<<([const] char [*]`